iml-manager-cli/src/main.rs

```diff
@@ -49,12 +49,12 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     tracing::subscriber::set_global_default(subscriber).unwrap();
 
-    dotenv::from_path("/var/lib/chroma/iml-settings.conf").expect("Could not load cli env");
-
     let matches = App::from_args();
 
     tracing::debug!("Matching args {:?}", matches);
 
+    dotenv::from_path("/var/lib/chroma/iml-settings.conf").expect("Could not load cli env");
```

Comment:
Why this change?

Reply:
Because application doesn't generate panic for `-h` request